studiopress/pattern-manager
# Convert `src` entry point files to TS/TSX

## Tasks


### Summary of changes

This PR extends the default `wp-scripts` webpack config to use TypeScript for the entry point in `app` and `editor`.

Admittedly, there is no huge gain in doing this now, but it was the intent of the initial TypeScript refactor to use TypeScript for all `src` files. Doing so would allow us to strictly enforce TypeScript-only for anything that will be transpiled to `build`.

### How to test

1. Checkout the branch
2. Make sure `app` and `editor` both render with `npm run dev`
